Picturepark.ContentPortal.Demo/Picturepark.ContentPortal.Demo/ClientApp/src/app/utilities/translations.ts

```diff
@@ -0,0 +1,105 @@
+export let Translations = {
+  Dashboard: {
+    Title: {
+      en: 'Welcome to the FOFA Content Portal!',
+      de: 'Willkommen im FOFA Content Portal!',
+    },
+    DemoInfo: {
+      en: 'More info about this demo',
+      de: 'Weitere infos zu dieser demo',
+    },
+    Content: {
+      en: `Our association is all about sharing knowledge, logistics and resources to promote organic farming - to both
+      farmers and the public. As a valued member of our organization, you can access carefully researched and approved
+      content for reuse via this Content Portal.`,
+      de: `In unserem Verein geht es darum, Wissen, Logistik und Ressourcen zu teilen, um den ökologischen Landbau zu 
+      fördern - für beide Landwirte und die Öffentlichkeit. Als geschätztes Mitglied unserer Organisation können Sie auf 
+      sorgfältig recherchierte und genehmigte Daten zugreifen Inhalte zur Wiederverwendung über dieses Inhaltsportal.`,
+    },
+    Contact1: {
+      en: 'If you would like to contribute your ideas, please',
+      de: 'Wenn Sie Ihre Ideen einbringen möchten, bitte',
+    },
+    Contact2: {
+      en: 'contact us',
+      de: 'kontaktiere uns',
```

Comment:
kontaktieren Sie uns - we always use the formal

Reply:
Done